committee-code-of-conduct/governance/onboarding-offboarding.md

```diff
@@ -0,0 +1,41 @@
+# Onboarding and offboarding new Code of Conduct Committee members
+
+Different actions on this list must be carried out by different members: 
+
+- **Outgoing members:** members who are ending their term
+- **Incoming members:** members beginning their term
+- **Carryover members:** members mid-way through their term
+
+## Permissions 
+
+### Slack
+
+**Who executes:** When offboarding, outgoing members must remove themselves from Slack channels. When onboarding, carryover members must add incoming members.
+
+- [ ] Code of conduct committe Slack channel on `kubernetes.slack.com`
+- [ ] Code of conduct sync Slack channel on `cloud-native.slack.com`
+
+### Kubernetes/community permissions and google permissions
+
+Permissions in this 
+
+**Who executes:** Carryover members should ensure that outgoing members are removed and incoming members are invited. 
+
+- [ ] Update `kubernetes/community` [`sigs.yaml`](https://github.com/kubernetes/community/blob/master/sigs.yaml) 
+- [ ] Update `kubernetes/k8s.io` [`groups.yaml`](https://github.com/kubernetes/k8s.io/blob/ca864e62da1d38832d3026da3021f0cbf156dd1d/groups/groups.yaml)
+
+    > **Note:** This permission controls access to the mailing list and Google Drive!
```

Comment:
hah! yea, we learned the hard way this time.

Could we add a note explicitly saying that direct changes to mailing list and drive will be reverted by the automation?